damus/Models/HomeModel.swift

```diff
@@ -579,6 +579,15 @@ func process_metadata_event(profiles: Profiles, ev: NostrEvent) {
     }
     
     notify(.profile_updated, ProfileUpdate(pubkey: ev.pubkey, profile: profile))
```

Comment:
this line needs to be removed, we don't need to update profiles twice

Reply:
Fixed. :)